### ISSUE TITLE
Improve sort option with a template

### ISSUE THREAD

**Issue by SergioRequena** [Metron-Project/metron-tagger#159]
First of all, **thank you very much** for all the work you’ve done on this project.

Now that there’s a first version of `MetronInfo.xml`, my request would be to allow customization of the sort option with a template, similar to what already exists for the file name.
In my case, I like to include the `startYear` in the directory name of a volume, and now that we have this information in the MetronInfo file, it should be possible. Apologies if this feature already exists, but I haven’t been able to find it.

Thank you.

**Comment by bpepple** [comment]
No, it currently doesn't since it could merge multiple series that were published in the same year, like the 2 Alien series published by Marvel last year, which could cause the user to accidentily overwrite an existing file. That was the reasoning behind using a volume number, that each series is separated.

Anyway... I'm not opposed to adding this option, tho I'm not 100% sure a template option would be needed, maybe just a boolean setting to either use volume or start year when sorting, and then add some code to change the filename of a file if it already exists in the directory it's being moved to.

**Comment by SergioRequena** [comment]
You're right, I hadn't considered that possibility. Even so, I still believe the template would provide users with customization options for organizing their collection.

In my case: `{publisher}/{startYear}/{series} {volume}`

**Comment by bpepple** [comment]
So... I've been wood-shedding different methods to see if this is a worth implementing or not, and I've come to the conclusion _it's not_.

Looking at the various bits of metadata available, and the only one that seems useful that we don't already use, is the start year. So, implementing a template style system doesn't seem terribly useful for just one additional piece of metadata.

I'm also a big proponent of not adding additional options if there isn't a _very_ compelling reason. The one big item that the sorting structure should support is not having a value that can result in the same path for multiple series (which is why volume is used, since it's a distinct value in the database, unlike series year).

If the user ___really___ wants to use a different sorting method, I'd suggest they look at a program like [Perdoo](https://github.com/Buried-In-Code/Perdoo) which might suit the user case better.

**Comment by SergioRequena** [comment]
I agree with you that this feature could cause problems when tow series realese an a same year.

I've beeing using the default sorting method and I think it works well. The only problem is that when I open a directory like Marvel or DC, I have more than 1500 folders and it is slow.
